src/components/NewWorkspaceModal.js

```diff
@@ -69,6 +71,10 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
     ...(requiredAuthDomain ? [requiredAuthDomain] : [])
   ])
 
+  const loadAlphaRegionalityUser = reportErrorAndRethrow('Error loading user group membership')(async () => {
```

Comment:
I would suggest making this error more specific so we know that it was encountered when trying to obtain the regionality group membership (`Error loading regionality group membership`). I think you copied this from my code in List.js, so I will make mine more specific as well. :)

Reply:
Done! And thank you for being the first example of this pattern